_component_design/checkboxes.md

```diff
@@ -67,42 +61,51 @@ allow the user to select zero, one, or several items.
   you to violate this guideline, but do try to keep choices visible whenever
   possible.
 
-### Interaction with checkboxes
+### Problems solved by checkboxes
 
-* Allow users to select an option by clicking on either the button, the box
-  itself or the label; a bigger target is faster to click.
+Checkboxes alllow users needs to make a selection in a form:
```

Comment:
Typos/grammar: Checkboxes allow users to make selections in a form, as follows:

Reply:
updated.